src/cc/tests/integtest.cc

```diff
@@ -83,23 +93,33 @@ QFSTest::TearDown()
 
 }
 
-string
-QFSTestUtils::WriteTempFile(const string& data)
+int
+QFSTestUtils::CreateTempFile(string* path)
 {
      string filename = QFSTestUtils::kTestHome + "/tmp_file.XXXXXX";
      int fd = mkstemp(const_cast<char*>(filename.c_str()));
      EXPECT_NE(-1, fd) << "Could not mkstep: " << strerror(errno);
```

Comment:
Yep. It's common to provide custom failure messages like this with Google Test. See the [documentation](https://github.com/google/googletest/blob/master/googletest/docs/V1_7_Primer.md#assertions) for more information. It's useful sometimes to have more context on why a failure occurred for debugging and whatnot.